src/TestFramework/Coverage/CoveredTraceProvider.php

```diff
@@ -46,10 +45,6 @@
  */
 final class CoveredTraceProvider implements TraceProvider
 {
-    /**
-     * @param PhpUnitXmlCoverageTraceProvider|TraceProvider $primaryTraceProvider
-     * @param BufferedSourceFileFilter|FileFilter $bufferedFilter
```

Comment:
This is unfortunate because without this doc comment one can no longer jump into the implementation with a single click.

Reply:
ok, reverted. Reported https://github.com/rectorphp/rector/issues/8633

let's see if this is a bug or if there are any arguments supporting this behavior